modules/check_in/app/sidekiq/check_in/travel_claim_submission_worker.rb

```diff
@@ -9,72 +9,109 @@ class TravelClaimSubmissionWorker
       redis_client = TravelClaim::RedisClient.build
       mobile_phone = redis_client.patient_cell_phone(uuid:)
       station_number = redis_client.station_number(uuid:)
+      facility_type = redis_client.facility_type(uuid:)
 
       logger.info({
-                    message: "Submitting travel claim for #{uuid}, #{appointment_date}, #{station_number}",
+                    message: "Submitting travel claim for #{uuid}, #{appointment_date}, " \
+                             "#{station_number}, #{facility_type}",
                     uuid:,
                     appointment_date:,
-                    station_number:
+                    station_number:,
+                    facility_type:
                   })
 
-      claim_number, template_id = submit_claim(uuid:, appointment_date:, station_number:)
+      claim_number, template_id = submit_claim(uuid:, appointment_date:, station_number:, facility_type:)
 
-      send_notification(mobile_phone:, appointment_date:, template_id:, claim_number:)
+      send_notification(mobile_phone:, appointment_date:, template_id:, claim_number:, facility_type:)
       StatsD.increment(STATSD_NOTIFY_SUCCESS)
     end
 
     def submit_claim(opts = {})
       check_in_session = CheckIn::V2::Session.build(data: { uuid: opts[:uuid] })
+
       claims_resp = TravelClaim::Service.build(
         check_in: check_in_session,
         params: { appointment_date: opts[:appointment_date] }
       ).submit_claim
 
-      handle_response(claims_resp:)
-    rescue Common::Exceptions::BackendServiceException => e
+      handle_response(claims_resp:, facility_type: opts[:facility_type])
+    rescue => e
       logger.error({ message: "Error calling BTSSS Service: #{e.message}" }.merge(opts))
-      StatsD.increment(STATSD_BTSSS_ERROR)
-      [nil, ERROR_TEMPLATE_ID]
+      if 'oh'.casecmp?(opts[:facility_type])
+        StatsD.increment(OH_STATSD_BTSSS_ERROR)
+        template_id = OH_ERROR_TEMPLATE_ID
+      else
+        StatsD.increment(CIE_STATSD_BTSSS_ERROR)
+        template_id = CIE_ERROR_TEMPLATE_ID
+      end
+      [nil, template_id]
     end
 
-    def handle_response(claims_resp:)
-      claim_number = claims_resp&.dig(:data, :claimNumber)&.last(4)
-      template_id =
-        case claims_resp&.dig(:data, :code)
-        when TravelClaim::Response::CODE_SUCCESS
-          StatsD.increment(STATSD_BTSSS_SUCCESS)
-          SUCCESS_TEMPLATE_ID
-        when TravelClaim::Response::CODE_CLAIM_EXISTS
-          StatsD.increment(STATSD_BTSSS_DUPLICATE)
-          DUPLICATE_TEMPLATE_ID
-        else
-          StatsD.increment(STATSD_BTSSS_ERROR)
-          ERROR_TEMPLATE_ID
-        end
+    # rubocop:disable Metrics/MethodLength
+    def handle_response(opts = {})
+      claim_number = opts[:claims_resp]&.dig(:data, :claimNumber)&.last(4)
+      code = opts[:claims_resp]&.dig(:data, :code)
+      facility_type = opts[:facility_type] || ''
+
+      statsd_increment, template_id = case facility_type.downcase
```

Comment:
nit-pick: instead of `statsd_increment`, can we pls name it `statsd_metric`? Atleast to me `statsd_increment` sounds like an integer value

Reply:
Ping me in Slack for a re-approval if this gets picked up :)

Reply:
updated in 81378401c68ea1aa17d71513b132cd8afcb93313